### DISCUSSION TITLE
What is Fiber?

### DISCUSSION THREAD

**Discussion by redbar0n** [frejs/fre#320] [Q&A] [answered]
The repo says fre is "for Fiber" and "with Fiber", but it doesn't link to what Fiber they are talking about.

Many things are called Fiber nowadays:

https://github.com/gofiber/fiber
https://en.wikipedia.org/wiki/Fiber_(computer_science)
https://github.com/fibjs/fibjs
https://wcandillon.github.io/react-native-fiber/

and googling it wasn't particularly helpful...

What is Fiber in the context of Fre?

A link in the readme would have been helpful.

**Comment by redbar0n** [comment, marked as answer]
I think I may have found it, at last...

https://www.velotio.com/engineering-blog/react-fiber-algorithm

But this is confusing.. Isn't Fre supposed to be a React _alternative_?

**Reply by yisar** [reply]
You should have found the answer. 
Fre's overall scheduling algorithm is the same as react. 
In the early days, they were called Fiber. Now React team prefers to be called concurrent mode.
